crypto/ec_extra/ec_asn1.c

```diff
@@ -533,3 +533,45 @@ int i2o_ECPublicKey(const EC_KEY *key, uint8_t **outp) {
   // Historically, this function used the wrong return value on error.
   return ret > 0 ? ret : 0;
 }
+
+static size_t EC_POINT_point2buf(const EC_GROUP *group, const EC_POINT *point,
+                                 point_conversion_form_t form,
+                                 unsigned char **pbuf, BN_CTX *ctx) {
+  size_t len;
+  unsigned char *buf;
+
+  len = EC_POINT_point2oct(group, point, form, NULL, 0, NULL);
+  if (len == 0) {
+    return 0;
+  }
+  if ((buf = OPENSSL_malloc(len)) == NULL) {
```

Comment:
nit: Assignments and conditions in an if statement in C, while valid, has always felt a bit like a dirty practice in the language. Not sure what our practice has been in the rest of the codebase, but it might be clearer to separate this out.